components/org.wso2.carbon.identity.api.server.workflow.management/org.wso2.carbon.identity.api.server.workflow.management.v1/src/main/resources/workflow.mgt.yaml

```diff
@@ -0,0 +1,126 @@
+openapi: 3.0.0
+info:
+  version: "v1"
+  title: 'Workflow Management API Definition'
+  description: 'This is the REST API for managing workflow requests'
+  contact:
+    name: WSO2
+    url: 'https://wso2.com/ciam-suite/private-ciam-cloud/b2b-ciam'
+    email: iam-dev@wso2.org
+  license:
+    name: Apache 2.0
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+servers:
+  - url: 'https://localhost:9443/workflow'
+
+security:
+  - OAuth2: []
+  - BasicAuth: []
+
+tags:
+  - name: workflow
+    description: Operations for the workflow requests.
+
+paths:
+  /workflow/{workflow-id}:
+    patch:
+      tags:
+        - me
+      summary: |
+        Update the workflow request status.
+      description: |
+        This API provides the capability to update the workflow request status.
+      parameters:
+        - name: workflow-id
+          in: path
+          description: ID of the workflow request.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/workflowApprovalPatchRequest'
+      responses:
+        '200':
+          description: Successful Response
+
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+components:
+  schemas:
+    Error:
+      type: object
+      required:
+        - code
+        - message
+      properties:
+        code:
+          type: string
+          example: UOM-00000
```

Comment:
is the correct error code prefix?

Reply:
fixed by : https://github.com/wso2/identity-api-server/pull/446/commits/3f0c88651577bd8b27c0f5d34ba6922881bcfe38